cabal-testsuite/PackageTests/RtsOptsClean/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```

Comment:
I think that file is no longer needed (with high enough cabal-version?).

Reply:
I can experiment and remove it. Would CI catch the problem if it exists?

Reply:
Most definitely yes. It even catches problems that don't exist (e.g., the Windows validated runs).

Reply:
Love it: the ultimate response of a sound type checker — "reject!"

Meanwhile, CI finished green without the file.